pkg/discovery/helper.go

```diff
@@ -51,6 +51,12 @@ type Helper interface {
 	APIGroups() []metav1.APIGroup
 }
 
+// DiscoveryInterface exposes functions for Kubernetes discovery
```

Comment:
s/DiscoveryInterface/ServerResourcesInterface

Also, this interface doesn't need to be exported given how you're using it - so can be `type serverResourcesInterface interface`